src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/Prisoner.kt

```diff
@@ -174,6 +174,8 @@ open class Prisoner {
     @Schema(description = "Actual Parole Date", example = "2023-05-01")
     var actualParoleDate: LocalDate? = null
 
+     @Schema(description = "current prison or outside with last movement information.", example = "Outside - released from Leeds")
+     var locationDescription: String? = null
```

Comment:
can't remember the prison API change - can this be nullable?

Reply:
If the prisoner is not `OUT` then no value will be passed, it is only added when the prisoner is outside